### DISCUSSION TITLE
The upcoming stimulus reflex and cable ready releases ⭐️

### DISCUSSION THREAD

**Discussion by omarluq** [joshleblanc/view_component_reflex#89] [Q&A]
Hey @joshleblanc, first of all a big thank you for you and everyone that has contributed to this project so far 👏🏼 
I gotta say VCR must be my favorite frontend lib for rails, my components never looked better. 
I was wondering if there are any concerns with the upcoming SR/CR releases, those releases has been in the works for a while and Im just worried about the upgrade strategy specially that I have an app in the works that will go to prod in 6 months and VCR are pretty much the lungs of my frontend stack at this point. 
Thank you.

**Comment by joshleblanc** [comment]
@omarluq The current pre-release of StimulusReflex (Which we depend on in the current release of VCR) should be fairly 1:1 with any upcoming release of SR. (Or at least, that was the plan before leastbad left the project).

I fully intend to update VCR to account for whatever breaking changes come up from the release. I don't expect anything too dramatic.

**Comment by omarluq** [comment]
@joshleblanc that is very reassuring! thanks. 
How would you feel if I made some videos to show people how to build some complex components with VCR such as step wizards, pop-up modals, slide panels...etc? wanna make sure you are ok with it before I start putting anything out there
